code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -2477,19 +2424,6 @@
 	reagents.add_reagent("imidazoline", 3)
 	bitesize = 2
 
-/obj/item/weapon/reagent_containers/food/snacks/superbiteburger

```

Comment:
Why?

Reply:
That's the one burger that's larger than a person.

Reply:
Also: meant to only remove the recipe. One moment.